fcm.go

```diff
@@ -118,7 +120,64 @@ func (fcm *FCM) sendFirebaseRequest(payload Message) (*Response, error) {
 	}
 
 	// create request
-	req, err := http.NewRequest("POST", fcmEndpoint, bytes.NewBuffer(data))
+	req, err := http.NewRequest("POST", fcmSendEndpoint, bytes.NewBuffer(data))
+	if err != nil {
+		return nil, err
+	}
+
+	// add headers
+	req.Header.Set("Authorization", fmt.Sprintf("key=%s", fcm.app.apiKey))
+	req.Header.Set("Content-Type", "application/json")
+
+	// TODO use fcm.app.client if possible (not working actually)
+	client := &http.Client{}
+
+	// execute request
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
```

Comment:
it is better to discard `resp.Body` when don't read the body
or just `defer io.Copy(ioutil.Discard, resp.Body)` after `defer resp.Body.Close()`